Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.10
+
+RUN go get github.com/kardianos/govendor
+RUN mkdir -p /go/src/github.com/kochman/hotshots
+WORKDIR /go/src/github.com/kochman/hotshots
+COPY vendor/vendor.json ./vendor/
+RUN govendor sync
+
+COPY . /go/src/github.com/kochman/hotshots
+RUN go install github.com/kochman/hotshots
+
+ENV HOTSHOTS_DIR /var/hotshots
```

Comment:
This is predefined in code, we don't need to set the env var

Reply:
Also you need to make the directory /var/hotshots and make sure permissions work for hotshots, otherwise it won't start.

Reply:
Okay, remove that line and add some form of documentation and I'll merge